src/Helper/GitHelper.php

```diff
@@ -140,7 +142,15 @@ public function restoreStashedBranch()
      */
     public function isGitDir($requireRoot = true)
     {
-        $directory = $this->getGitDir();
+        try {
+            $directory = $this->getGitDir();
+        } catch (\RuntimeException $e) {
+            if (128 === $e->getCode() && 'fatal: Not a git repository (or any of the parent directories): .git' === $e->getMessage()) {
```

Comment:
That was my first thought @GrahamCampbell, but I don't know which is the git's mapping for exit codes.